README.md

```diff
@@ -5,24 +5,19 @@ Can be used in combination with [Release Drafter](https://github.com/toolmantim/
 
 ## Usage
 
-Add `.github/main.workflow` with the following:
+Add `.github/workflows/pr-labeler.yml` with the following:
 
-```
-workflow "Add label to PR" {
-  on = "pull_request"
-  resolves = "PR Labeler"
-}
-
-action "PR opened filter" {
-  uses = "actions/bin/filter@master"
-  args = "action opened"
-}
-
-action "PR Labeler" {
-  needs = "PR opened filter"
-  uses = "TimonVS/pr-labeler@master"
-  secrets = ["GITHUB_TOKEN"]
-}
+```yml
+name: Label PRs
+on: [pull_request]
```

Comment:
As I was trying to set this up as well, I found the docs show that you can add a `types` to the `pull_request` event removing the need for the `if` below. [Events that Trigger Workflows: Pull Request](https://help.github.com/en/articles/events-that-trigger-workflows#pull-request-event-pull_request) It seemed to work, but I got a failure because of the github token (hence why I'm on this PR) :)

```yml
name: PR Labeler
on: 
  pull_request:
    types: [opened]

jobs:
  pr-labeler:
    runs-on: ubuntu-latest
    steps:
      - uses: TimonVS/pr-labeler@master
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Just pushed an update to my branch and it didn't run the `pr-labeler` workflow using the above.

Reply:
I tried the config above in https://github.com/TimonVS/pr-labeler-test and it seems to work fine for me. Shall we use that one in the readme? :)